pkg/await/awaiters.go

```diff
@@ -173,9 +177,9 @@ var awaiters = map[string]awaitSpec{
 		awaitCreation: untilCoreV1PersistentVolumeClaimBound,
 	},
 	coreV1Pod: {
-		// NOTE: Because we replace the Pod in most situations, we do not require special logic for
-		// the update path.
-		awaitCreation: func(c createAwaitConfig) error { return makePodInitAwaiter(c).Await() },
+		awaitCreation: awaitPodInit,
+		awaitRead:     awaitPodRead,
```

Comment:
It's true in most cases, but IIRC, it wasn't working quite right as I was testing individual Pod updates. Seemed like it made sense to standardize since the change was trivial.